### PR TITLE
SecHUD markers

### PR DESCRIPTION
**Pull request by TheChosenEvilOne**
## About The Pull Request
Adds markers you can create when wearing SecHUD glasses, only visible to other people with SecHUD.
ctrl+mmb makes an alert marker and alt+mmb makes a question marker.

## Why It's Good For The Game
Makes it easier for sec to coordinate quickly.

## Changelog
:cl: tceo, sprites by Twaticus
add: When wearing SecHUD glasses you can now create markers with ctrl + MMB (alert marker) and alt + MMB (question marker).
/:cl:

### CODE REVIEW

**Comment by Pugie** [comment]
toxic LoL PR

**Comment by TheChosenEvilOne** [comment]
> toxic LoL PR

yes

**Comment by EdgeLordExe** [comment]
add a sound alert that plays to nearby sechud users when placing/removing a marker

**Comment by TheChosenEvilOne** [comment]
> add a sound alert that plays to nearby sechud users when placing/removing a marker

I have been thinking about it for a bit, and the only reasonable way of playing audio I can think of is by checking the distance to every person wearing sechud. I guess Ill have to look into doing that.

File: code/modules/clothing/glasses/hud.dm [hunk 1]
```diff
@@ -102,6 +102,43 @@
...
+		return FALSE
+	RegisterSignal(user, COMSIG_MOB_MIDDLECLICKON, .proc/create_ping)
+
+/obj/item/clothing/glasses/hud/security/dropped(mob/living/carbon/human/user)
+	. = ..()
+	if (!.)
+		return FALSE
+	UnregisterSignal(user, COMSIG_MOB_MIDDLECLICKON)
+
+/obj/item/clothing/glasses/hud/security/proc/create_ping(datum/source, atom/A, params)
```
Comment (Canohaaaaaaa) [inline code/modules/clothing/glasses/hud.dm:117]: `SIGNAL_HANDLER`

File: code/modules/clothing/glasses/hud.dm [hunk 2]
```diff
@@ -102,6 +102,43 @@
...
+	var/list/modifiers = params2list(params)
+	var/mob/living/L = source
```
Comment (Canohaaaaaaa) [inline code/modules/clothing/glasses/hud.dm:119]: Descriptive var name

**Review by Canohaaaaaaa — CHANGES_REQUESTED** [review]

**Pull request merged**